dbms/src/Flash/Pipeline/Schedule/Events/Event.cpp

```diff
@@ -235,7 +235,9 @@ void Event::switchStatus(EventStatus from, EventStatus to)
         magic_enum::enum_name(to),
         magic_enum::enum_name(status.load()));
 
+#ifndef NDEBUG
     LOG_DEBUG(log, "switch status: {} --> {}", magic_enum::enum_name(from), magic_enum::enum_name(to));
+#endif // !NDEBUG
```

Comment:
I think debug level log is acceptable.

Reply:
Info level should be the default log level in the future. At that time `#ifndef NDEBUG` is useless. It's easier to change the config instead of rebuilding the tiflash in debug mode.

Reply:
Currently, the info level is only set under the cloud, and most user environments are at the debug level.
Or change the log level to trace? Because these logs are not helpful in troubleshooting in the user environment, they are only used for internal testing

Reply:
Looks good to me.

Reply:
reverted to the origin code.